YoutubeExplode/YoutubeClient.cs

```diff
@@ -75,6 +75,8 @@ public partial class YoutubeClient
 
             httpClient.DefaultRequestHeaders.Add("User-Agent",
                 "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.163 Safari/537.36");
+            httpClient.DefaultRequestHeaders.Add("x-youtube-client-name", "56");
+            httpClient.DefaultRequestHeaders.Add("x-youtube-client-version", "20200911");
```

Comment:
These headers should be added only to the request that calls the search api, if you put them in the other requests they will fail; try to run the unit tests.